src/pipelines/aws/aws-pipeline.ts

```diff
@@ -201,6 +236,28 @@ export default class AWSPipeline implements Pipeline {
       `${outputFolder}/${outputObject}`
     );
     if (!s3Status) return '';
+    if (variables ? variables['QVBR'] : undefined) {
```

Comment:
I think it would be better to check the 'probeBitrates' flag instead of checking the existance of QVBR variable here.